### PR TITLE
Misleading file extension of course enroll sample spreadsheet #5385

### LINKED ISSUE #5385: Misleading file extension of course enroll sample spreadsheet
On the course enrollment page, the sample spreadsheet will be downloaded as `Course Enroll Sample Spreadsheet.csv`, even though the enrolment page does not accept comma separated values.

Users actually need to use Excel (and perhaps other similar softwares) for the copy and paste to work. Text editors will not work.

Maybe we can save the file as `.xls` or `.ods` (openDocument spreadsheet) to avoid confusion.

### PR DESCRIPTION
**Pull request by YongJieYongJie**
Fixes #5385

### CODE REVIEW

File: src/test/java/teammates/test/pageobjects/AppPage.java [hunk 1]
```diff
@@ -890,6 +890,16 @@ public AppPage verifyHtmlMainContent(String filePath) throws IOException {
     }
     
     /**
+     * Verifies that the title of the loaded page is the same as {@code expectedTitle}
+     * @param expectedTitle
+     * @return The page (for chaining method calls).
+     */
+    public AppPage verifyTitle(String expectedTitle) {
+        assertEquals(expectedTitle, browser.driver.getTitle());
+        return this;
```
Comment (wkurniawan07) [inline src/test/java/teammates/test/pageobjects/AppPage.java]: You don't really need to `return this;`. It doesn't really make sense for any verification method to have a return value.

P.S I know the others have, and I'm quite itching to fix them :p

File: src/main/webapp/jsp/instructorCourseEnroll.jsp [hunk 2]
```diff
@@ -50,7 +50,7 @@
             <li><span class="moreInfoPointTitle">Sample spreadsheet</span>
                 <div class="moreInfoPointDetails">
                     If you are not sure about the format of the spreadsheet, 
-                    <a id ="spreadsheet_download" href="/files/Course%20Enroll%20Sample%20Spreadsheet.csv">here</a> is a sample file.
+                    <a id ="spreadsheet_link" href="https://docs.google.com/spreadsheets/d/1jODYiO_TcSoQ82t4E9rX-kfn1l7sEhb810rdlVJxHo0/edit?usp=sharing">here</a> is a sample file.
```
Comment (wkurniawan07) [inline src/main/webapp/jsp/instructorCourseEnroll.jsp]: To make it conforming to the HTML guide for naming (which is poorly enforced, but let's clean up when we can), use `spreadsheet-link`. Also, get rid of the space between `id` and `=`.

**Comment by wkurniawan07** [comment]
Also, I think there's at least an item in the [troubleshooting guide](https://github.com/TEAMMATES/teammates/blob/master/devdocs/troubleshooting-guide.md) that's made irrelevant by this change. Do identify and remove it (or them).

**Comment by YongJieYongJie** [comment]
@wkurniawan07 Made the changes as requested!

**Comment by wkurniawan07** [comment]
Good job @YongJieYongJie 👍 once this gets approval for merge, you can do it yourself. Don't forget to choose "squash and merge", and name the merge commit "[1234] Issue Title".

**Pull request merged**